packages/rancher-logging/generated-changes/overlay/templates/loggings/rke/logging-rke.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if .Values.additionalLoggingSources.rke.enabled }}
+{{- $containers := printf "%s/%s" .Values.global.dockerRootDirectory "containers" -}}
```

Comment:
Does this not need a default in case .Values.global.dockerRootDirectory is empty similar to above usages?

Reply:
No because `$containers` is only used within protected blocks within this file.
```yaml
{{- if .Values.global.dockerRootDirectory }}
# foo
{{- end }}
```
However, it might be a good idea to set a default anyway to prevent mishaps in the future. Thoughts?

Reply:
Thought about this some more and asked @paynejacob. Going to resolve. I think pre-optimizing isn't necessary here.